### PR TITLE
Load Balancing and Closing the Farm C++ Solutions

### PR DESCRIPTION
**Pull request by maggie-j-liu**
I added a C++ solution to Load Balancing (Feb 2016 Bronze) and a C++ solution using dfs (the official solution uses dsu) to Closing the Farm (Open 2016 Silver).

### CODE REVIEW

**Review by frodakcin — APPROVED** [review]
Clean. Unfortunately this will merge conflict with https://github.com/cpinitiative/usaco-guide/pull/701 uhh we'll deal with that later

**Comment by frodakcin** [comment]
I'll wait for a couple more reviews before merging.

**Review by dongliu0426 — APPROVED** [review]

**Review by Farrius — APPROVED** [review]

**Pull request merged**
